Tests/Tests/RouterSpec.swift

```diff
@@ -217,6 +260,48 @@ class RouterSpec: QuickSpec {
                 }
             }
 
+            context("when calling isDisplayingTooltip") {
+
+                let tooltipTargetView = UIView(frame: CGRect(x: 100, y: 100, width: 10, height: 10))
+                let tooltip = TestHelpers.generateTooltip(id: "test")
+                let imageBlob: Data! = UIImage(named: "test-image", in: .unitTests, with: nil)?.pngData()
+
+                beforeEach {
+                    tooltipTargetView.accessibilityIdentifier = TooltipViewIdentifierMock
+                    window.addSubview(tooltipTargetView)
+                }
+
+                afterEach {
+                    tooltipTargetView.removeFromSuperview()
+                    window.findTooltipView()?.removeFromSuperview()
+                }
+
+                it("will return true when tooltip displayed") {
+                    router.displayTooltip(tooltip, targetView: tooltipTargetView,
+                                          identifier: TooltipViewIdentifierMock,
+                                          imageBlob: imageBlob,
+                                          becameVisibleHandler: { _ in },
+                                          confirmation: true,
+                                          completion: { _ in })
+                    expect(window.findTooltipView()).toEventuallyNot(beNil())
+                    expect(router.isDisplayingTooltip(with: TooltipViewIdentifierMock)).to(beTrue())
+                }
+
+                it("will return false when tooltip is not displayed") {
+                    expect(router.isDisplayingTooltip(with: "test")).to(beFalse())
+                }
+
```

Comment:
please add one more case when tooltip is displayed but the function is called with another identifier.

Reply:
Addded here [3cee1ec](https://github.com/rakutentech/ios-inappmessaging/pull/275/commits/3cee1ec04fdbedfab437064f59c968abeb121232).